File: python/problem-001/main.py
def MultiplesOf3And5(limit):
  total = 0
  for number in range(limit):
    if number % 3 == 0 or number % 5 == 0 : 
      total += number
  return total

if __name__ == "__main__" :
  number = 1000
  result = MultiplesOf3And5(number)
  print(f"The sum of all multiples of 3 or 5 below {number} is: {result}")

File: c/problem-001/main.c
#include <stdio.h>

long long MultiplesOf3And5(int limit) {
  long long total = 0;
  for (int number = 0; number < limit; ++number) {
    if (number % 3 == 0 || number % 5 == 0) {
      total += number;
    }
  }
  return total;
}

int main(void) {
  printf("%lld\n", MultiplesOf3And5(1000));
  return 0;
}
